server/cmd/mmctl/commands/user.go

```diff
@@ -1000,3 +1030,30 @@ func demoteUserToGuestCmdF(c client.Client, _ *cobra.Command, userArgs []string)
 
 	return errs.ErrorOrNil()
 }
+
+func ssoAuthDataResetCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	includeDeleted, _ := cmd.Flags().GetBool("include-deleted")
+	dryRun, _ := cmd.Flags().GetBool("dry-run")
+	confirmed, _ := cmd.Flags().GetBool("yes")
+	userIDs, _ := cmd.Flags().GetStringSlice("users")
```

Comment:
I wonder if the command should be safe by default, i.e., it should only run on a selected number of users. Running it on all users would need to be forced via `--all`.